Dockerfile.cmd

```diff
@@ -14,20 +14,23 @@
 
 FROM open-match-base-build as builder
 
-WORKDIR /go/src/open-match.dev/open-match/cmd/minimatch/
-RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo .
+WORKDIR /go/src/open-match.dev/open-match
+
+ARG IMAGE_TITLE
+
+RUN make "build/cmd/${IMAGE_TITLE}"
 
 FROM gcr.io/distroless/static:nonroot
 WORKDIR /app/
-COPY --from=builder --chown=nonroot /go/src/open-match.dev/open-match/cmd/minimatch/minimatch /app/
 
-ENTRYPOINT ["/app/minimatch"]
+COPY --from=builder --chown=nonroot "/go/src/open-match.dev/open-match/build/cmd/${IMAGE_TITLE}/" "/app/"
```

Comment:
Is there a way to not make this `run`?

Reply:
This seems to be the most reasonable way.  Entrypoint doesn't use parameters from the build.